.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: Falco CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install -U pip
+        pip install -r requirements-dev.txt
+    - name: Lint
+      run: |
+        pip install flake8 black isort
```

Comment:
Are step contexts isolated?

Why do we need to reinstall flka8 etc. here if we already installed requirements-dev during the above step?

Reply:
I think they are not, I used [this file](https://github.com/actions/starter-workflows/blob/master/ci/python-package.yml) as guideline; I'll cleanup

Reply:
https://github.com/falcosecurity/client-py/pull/31